netrd/distance/onion_divergence.py

```diff
@@ -59,10 +61,8 @@ def dist(self, G1, G2, dist='lccm'):
 
         """
         # take the simple graph version
-        G1_simple = ensure_undirected(G1)
-        G2_simple = ensure_undirected(G2)
-        G1_simple = ensure_unweighted(G1_simple)
-        G2_simple = ensure_unweighted(G2_simple)
+        G1_simple = G1
```

Comment:
Do we need to rename these args?

Reply:
I think I would prefer keeping the `G1_simple` names and making them copies of `G`. I never have a great feel for when the underlying graph gets modified, but I think right now we'll kill the self-loops in the existing graph after calling `dist`.